### ISSUE TITLE
pressIt() tests only work if you use ASCII values 

### ISSUE THREAD

**Issue by NathanB09** [learn-co-curriculum/js-jquery-event-listeners-readme#26]
function pressIt() {
  $("#typing").on("keydown", function(e) {
    let ketPressed = e.key
    if (keyPressed === "g") {
      alert("You have pressed the G key!")
    }
  })
}

The above code works, but the tests do not acknowledge that. I found this way of coding it to be much easier as you do not need to know the ASCII values.

**Comment by justbennett** [comment]
I agree the test should be adjusted, but you might want to add `|| keyPressed === 'G'` . I tried `e.code === 'KeyG'` but that didn't seem to work.
I'm just going by results from [keycode.info](http://keycode.info/). Uppercase and lowercase 'g' return different results.
